website/versioned_docs/version-22.3/SnapshotTesting.md

```diff
@@ -136,7 +136,38 @@ From here you can choose to update that snapshot or skip to the next:
 
 ![](/jest/img/content/interactiveSnapshotUpdate.gif)
 
-### Tests Should Be Deterministic
+## Best Practices
+
+Snapshots are a fantastic tool for identifying unexpected interface changes
+within your application – whether that interface is a UI, logs, or error
+messages; however, they're not a cure-all, and, as with any testing strategy,
+there are some best-practices you should be aware of, and guidelines you should
+follow, in order to use them effectively.
```

Comment:
this was a loooong sentence! Mind splitting it up? I think just swapping the `;` for a `.` should be enough 🙂